scripts/start

```diff
@@ -1,23 +1,27 @@
 #!/bin/bash
 
-cd $(dirname $0)/..
-
-exists() {
-    [ -e "$1" ]
-}
-
-if exists Artemis-modules-extra; then
-    ADDITIONAL_OPTIONS=--scale=karton-sqlmap=5
+if [[ $(uname -m) != "arm"* ]]; then
+    echo "Artemis-Extra-Modules are only x86 compatible yet"
```

Comment:
the correct name is Artemis-modules-extra (even if it's not the best one possible, it is as it is now and let's use proper version and capitalization everywthere)